aws_adfs/login.py

```diff
@@ -151,7 +155,7 @@ def _authenticate(config, username=None, password=None):
     del password
 
     # Decode the response
-    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
+    html = ET.fromstring(response.text, ET.HTMLParser())
```

Comment:
It didn't work at all as-is in Python 3; I am not sure about the charset issue that was reported with Py2.